com.unity.ml-agents/Runtime/ActionMasker.cs

```diff
@@ -19,6 +19,43 @@ internal ActionMasker(BrainParameters brainParameters)
             m_BrainParameters = brainParameters;
         }
 
+        /// <summary>
+        /// Sets an action mask for discrete control agents. When used, the agent will not be
+        /// able to perform the action passed as argument at the next decision. If no branch is
+        /// specified, the default branch will be 0. The actionIndex or actionIndices correspond
+        /// to the action the agent will be unable to perform.
+        /// </summary>
+        /// <param name="actionIndices">The indices of the masked actions on branch 0</param>
+        protected void SetActionMask(IEnumerable<int> actionIndices)
```

Comment:
These should be public, right? They were protected on the Agent, but now they need to be called from outside of the Agent.

Reply:
you are right